### PR TITLE
Page: add Go-to-top button

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/628
[Problem] No Go-to-top button in content example
[Solution] Add Go-to-top button to Page widget according to
   the guideline and implement it for Content.
[Remarks] On lists, the button does not work since it will be
   handled in https://github.com/Samsung/TAU/issues/627.
    
 Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: src/css/profile/wearable/changeable/common/button.less [hunk 1]
```diff
@@ -355,3 +355,10 @@ button.ui-btn, input.ui-btn {
 		margin-top: 0;
 	}
 }
+
+.ui-button-go-to-top {
+	width: 52 * @unit_base;
+	height: 52 * @unit_base;
+	background-image: url("./images/GoToTopButton/go_to_top_dark.png");
```
Comment (TomaszLukawskiSam) [inline src/css/profile/wearable/changeable/common/button.less]: I think we shouldn't create additional directory like 'GoToTopButton' to store images. This makes additional issues when images will be updated.

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

**Comment by grzegorz-czajkowski** [comment]
As we discussed, I am changing the approach to add the button to the page (instead of scrollview). This will allow us to handle click on the button for arclistview.

File: src/css/profile/wearable/changeable/common/button.less (hunk 1 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/common/button.less]: We are having images in dirs https://github.com/Samsung/TAU/tree/tau_1.0/src/css/profile/wearable/changeable/theme-changeable/images

**Comment by grzegorz-czajkowski** [comment]
Added commit to enable go top button by Page option.

File: src/css/profile/wearable/changeable/common/button.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/wearable/changeable/common/button.less]: I thought about not creating additional subdirectories `GoToTopButton`

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**